benchmark/src/main/java/ai/timefold/solver/benchmark/impl/statistic/subsingle/constraintmatchtotalbestscore/ConstraintMatchTotalBestScoreStatisticPoint.java

```diff
@@ -23,12 +23,8 @@ public long getTimeMillisSpent() {
         return timeMillisSpent;
     }
 
-    public String getConstraintPackage() {
```

Comment:
Should we first deprecate it before removing it?

Reply:
I'm assuming this is an 'internal' class and won't break the user's code. If so, please ignore my previous comment.

Reply:
Yes, this class is internal, so I just went ahead and removed it. The information is there in `ConstraintRef` anyway.